homeassistant/components/renson/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""The Renson integration."""
+from __future__ import annotations
+
+import renson_endura_delta.renson as renson
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+
+PLATFORMS = [
+    "sensor",
+    "binary_sensor",
+]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Renson from a config entry."""
+
+    renson_api: renson.RensonVentilation = renson.RensonVentilation(entry.data["host"])
+
+    connected = await hass.async_add_executor_job(renson_api.connect)
+
+    if not connected:
+        raise ConfigEntryNotReady
```

Comment:
```suggestion
    if not await hass.async_add_executor_job(renson_api.connect):
        raise ConfigEntryNotReady
```

Additionally, you might want to add a comment why in the raise, e.g.:

`raise ConfigEntryNotReady("Because the moon is aligned with Mars")` or something 😉